### DISCUSSION TITLE
Where can I find the file for this url (https://www.sqlalchemy.org/develop.html)?

### DISCUSSION THREAD

**Discussion by ybjeon01** [sqlalchemy/sqlalchemy#12073] [Usage Questions] [answered]
Hi,

I'm reading this document before submitting a pull request related to a typo, and I noticed a minor typo in this document itself. However, I'm not sure where the file that generates this [develop.html](https://www.sqlalchemy.org/develop.html) is located in this repo. For example, under the Testing section, it says `py.test` instead of `pytest.

and Have a nice day!

**Comment by zzzeek** [comment, marked as answer]
that file is [here](https://github.com/sqlalchemyorg/sqlalchemyorg/blob/main/develop.html.mako)
